### PR TITLE
Update doc urls for /features + /learn

### PR DESCRIPTION
**Pull request by chibicode**
Updated documentation links for the features pages and learn pages. Also fixed some incorrect routes.

### CODE REVIEW

File: lib/docs/hash-map.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
     "with-url-object-1": "/api-reference/next/router#with-url-object",
     "router-events": "/api-reference/next/router#routerevents",
     "shallow-routing": "/routing/shallow-routing",
-    "useRouter": "/api-reference/next/router#userouter",
+    "userouter": "/api-reference/next/router#userouter",
```
Comment (chibicode) [inline lib/docs/hash-map.json]: Hashes are all lowercase - this was a typo.

File: lib/docs/hash-map.json [hunk 2]
```diff
@@ -17,7 +17,7 @@
     "dynamic-routing": "/routing/dynamic-routes",
     "populating-head": "/api-reference/next/head",
     "fetching-data-and-component-lifecycle": "/api-reference/data-fetching/getInitialProps",
-    "routing": "/api-routes/introduction",
+    "routing": "/routing/introduction",
```
Comment (chibicode) [inline lib/docs/hash-map.json]: This was a typo.

**Comment by zeit-team** [comment]
#### 📝Changed routes:


| [**…outing/dynamic-routing**](https://next-site-git-fork-chibicode-update-doc-urls.zeit.now.sh/learn/basics/clean-urls-with-dynamic-routing/dynamic-routing) | [**…ynamic-routing/finally**](https://next-site-git-fork-chibicode-update-doc-urls.zeit.now.sh/learn/basics/clean-urls-with-dynamic-routing/finally) |
|:-:|:-:|
|<a href="https://next-site-git-fork-chibicode-update-doc-urls.zeit.now.sh/learn/basics/clean-urls-with-dynamic-routing/dynamic-routing">Screenshot of /learn/basics/clean-urls-with-dynamic-routing/dynamic-routing</a><br /><sup><a href="https://images.weserv.nl?url=https://microlink.nyc3.digitaloceanspaces.com/nYmD5h8Fyexc6_pilBxR3D7U0qY_">(view full size)</a> |<a href="https://next-site-git-fork-chibicode-update-doc-urls.zeit.now.sh/learn/basics/clean-urls-with-dynamic-routing/finally">Screenshot of /learn/basics/clean-urls-with-dynamic-routing/finally</a><br /><sup><a href="https://images.weserv.nl?url=https://microlink.nyc3.digitaloceanspaces.com/OI9HOjFeKDgQzVWHNZA6s_wCeoE_">(view full size)</a> |



| [**…mic-pages/passing-data**](https://next-site-git-fork-chibicode-update-doc-urls.zeit.now.sh/learn/basics/create-dynamic-pages/passing-data) | [**…namic-pages/use-router**](https://next-site-git-fork-chibicode-update-doc-urls.zeit.now.sh/learn/basics/create-dynamic-pages/use-router) |
|:-:|:-:|
|<a href="https://next-site-git-fork-chibicode-update-doc-urls.zeit.now.sh/learn/basics/create-dynamic-pages/passing-data">Screenshot of /learn/basics/create-dynamic-pages/passing-data</a><br /><sup><a href="https://images.weserv.nl?url=https://microlink.nyc3.digitaloceanspaces.com/w__lX786Ilz0s8elBbVPasXp4pc_">(view full size)</a> |<a href="https://next-site-git-fork-chibicode-update-doc-urls.zeit.now.sh/learn/basics/create-dynamic-pages/use-router">Screenshot of /learn/basics/create-dynamic-pages/use-router</a><br /><sup><a href="https://images.weserv.nl?url=https://microlink.nyc3.digitaloceanspaces.com/FWoMxAAJByVIGzvBD8AcvPA4PpI_">(view full size)</a> |



| [**…/deploying-to-zeit-now**](https://next-site-git-fork-chibicode-update-doc-urls.zeit.now.sh/learn/basics/deploying-a-nextjs-app/deploying-to-zeit-now) | [**…g-a-nextjs-app/finally**](https://next-site-git-fork-chibicode-update-doc-urls.zeit.now.sh/learn/basics/deploying-a-nextjs-app/finally) |
|:-:|:-:|
|<a href="https://next-site-git-fork-chibicode-update-doc-urls.zeit.now.sh/learn/basics/deploying-a-nextjs-app/deploying-to-zeit-now">Screenshot of /learn/basics/deploying-a-nextjs-app/deploying-to-zeit-now</a><br /><sup><a href="https://images.weserv.nl?url=https://microlink.nyc3.digitaloceanspaces.com/7n1I2aodCd03riW2QUCUofVni9Y_">(view full size)</a> |<a href="https://next-site-git-fork-chibicode-update-doc-urls.zeit.now.sh/learn/basics/deploying-a-nextjs-app/finally">Screenshot of /learn/basics/deploying-a-nextjs-app/finally</a><br /><sup><a href="https://images.weserv.nl?url=https://microlink.nyc3.digitaloceanspaces.com/IoOzUBqk5hqonXcc4OZeq9XkVaM_">(view full size)</a> |



And 9 other routes:
- [**/learn/basics/fetching-data-for-pages/finally**](https://next-site-git-fork-chibicode-update-doc-urls.zeit.now.sh/learn/basics/fetching-data-for-pages/finally)
- [**/learn/basics/navigate-between-pages/simple-but-powerful**](https://next-site-git-fork-chibicode-update-doc-urls.zeit.now.sh/learn/basics/navigate-between-pages/simple-but-powerful)
- [**/learn/excel/automatic-static-optimization**](https://next-site-git-fork-chibicode-update-doc-urls.zeit.now.sh/learn/excel/automatic-static-optimization)
- [**/learn/excel/lazy-loading-components/finally**](https://next-site-git-fork-chibicode-update-doc-urls.zeit.now.sh/learn/excel/lazy-loading-components/finally)
- [**/learn/excel/lazy-loading-components/hello-dynamic-components**](https://next-site-git-fork-chibicode-update-doc-urls.zeit.now.sh/learn/excel/lazy-loading-components/hello-dynamic-components)
- [**/learn/excel/lazy-loading-components/lazy-loading**](https://next-site-git-fork-chibicode-update-doc-urls.zeit.now.sh/learn/excel/lazy-loading-components/lazy-loading)
- [**/learn/excel/lazy-loading-modules/lazy-loading**](https://next-site-git-fork-chibicode-update-doc-urls.zeit.now.sh/learn/excel/lazy-loading-modules/lazy-loading)
- [**/learn/excel/static-html-export/setup**](https://next-site-git-fork-chibicode-update-doc-urls.zeit.now.sh/learn/excel/static-html-export/setup)
- [**/learn/excel/static-html-export/thats-it**](https://next-site-git-fork-chibicode-update-doc-urls.zeit.now.sh/learn/excel/static-html-export/thats-it)



Commit e99f7b2f14dcc22b02e37f9d4f81beaec0f0b321 (https://next-site-git-fork-chibicode-update-doc-urls.zeit.now.sh).

File: pages/learn/basics/deploying-a-nextjs-app/deploying-to-zeit-now.mdx [hunk 3]
```diff
@@ -23,7 +23,7 @@ That's it!
 Your application is deployed to production.
 You'll see a URL printed in your terminal that your application is accessible at.
 
-To read more about what to do next, please visit [▲ZEIT Now's documentation](https://zeit.co/docs#related).
+To read more about what to do next, please visit [▲ZEIT Now's documentation](https://zeit.co/docs).
```
Comment (chibicode) [inline pages/learn/basics/deploying-a-nextjs-app/deploying-to-zeit-now.mdx]: Should go to the top of the documentation page, so removed `#related`.

File: components/css-in-js/links.js [hunk 4]
```diff
@@ -43,7 +43,7 @@ export default () => (
           your application. Seamlessly add and remove components without constantly ballooning
           stylesheet size and complexity.
         </p>
-        <Button href="/docs#css-in-js" amp>
+        <Button href="/docs/basic-features/built-in-css-support" amp>
```
Comment (chibicode) [inline components/css-in-js/links.js]: This is for [this page](https://next-site-git-fork-chibicode-update-doc-urls.zeit.now.sh/features/css-in-js).

File: components/css-in-js/docs.js [hunk 5]
```diff
@@ -15,7 +15,7 @@ export default () => (
         </p>
       </div>
       <div>
-        <Button invert href="/docs#css-in-js" amp>
+        <Button invert href="/docs/basic-features/built-in-css-support" amp>
```
Comment (chibicode) [inline components/css-in-js/docs.js]: This is for [this page](https://next-site-git-fork-chibicode-update-doc-urls.zeit.now.sh/features/css-in-js).

**Pull request merged**